lib/pages/V1App.dart

```diff
@@ -511,34 +507,6 @@ class _V1AppState extends State<V1App> {
         child: Row(
           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
           children: <Widget>[
-            if (network)
-              Row(
-                mainAxisAlignment: MainAxisAlignment.spaceAround,
-                children: <Widget>[
-                  const Icon(
-                    Icons.link,
-                    color: Colors.grey,
-                  ),
-                  DropdownButtonHideUnderline(
-                    child: ButtonTheme(
-                      alignedDropdown: true,
-                      child: DropdownButton(
-                        dropdownColor: Colors.black,
-                        borderRadius: BorderRadius.circular(10),
-                        elevation: 1,
-                        value: selectVal,
-                        items: dropDownMenuItems.map((itemOne) {
-                          return DropdownMenuItem(
-                              enabled: false,
-                              value: itemOne,
-                              child: Text(itemOne));
-                        }).toList(),
-                        onChanged: (value) {},
-                      ),
-                    ),
-                  )
-                ],
-              ),
```

Comment:
Just a note: At some point we will need it when we support SX, so its fine that we remove it for now. We can always create a ticket to add it back in at a later point.